kgforge/specializations/stores/nexus/service.py

```diff
@@ -582,3 +587,27 @@ def to_resource(
         if not hasattr(resource, "id") and kwargs and 'id' in kwargs.keys():
             resource.id = kwargs.get("id")
         return resource
+
+
+def _error_message(error: Union[HTTPError, Dict]) -> str:
```

Comment:
Why do we make it a string? instead of passing the full dictionary?
Also, I have the feeling that this doesn't work, it does not return the complete error, and that is why I tried to change it in a [this PR ](https://github.com/BlueBrain/nexus-forge/pull/281/files)some time ago.

Reply:
Let merge this one here and look at the other PR

Reply:
OK